__tests__/external/ACSAdapter/AMSFileManager.spec.ts

```diff
@@ -57,20 +61,25 @@ describe('AMSFileManager', () => {
 
         const response = fileManager.getFileIds(metadata);
         expect(response).toStrictEqual(amsReferences);
+        expect(logger.startScenario).toBeCalledTimes(1);
     });
 
-    it('AMSFileManager.getFileIds() should return a nothing if invalid', async () => {
+    it('AMSFileManager.getFileIds() should return nothing if invalid', async () => {

```

Comment:
is this valid?, if something is invalid, isnt better to return an error code?

Reply:
It's valid. If there is no metadata, it should be a no-op. If there's `fileIds`, it means that there's an attachment then it should be parsed. If the parsing fails, it should only be logged then continue. We always have to ensure the FileManager does not throw any exception to ensure the conversation is flowing. If an exception was thrown then not caught, WebChat will stop working.